### PR TITLE
scan: implement user delegation status on validator index page

### LINKED ISSUE #1725: Implement user delegation status on validator index page
- design layout(I think it should place on top of the information box.)
- show current stake on that validator
- show unbending amount, completion time
- show reward balance
- implement UI for delegate button, undelegate button, withdraw reward button
- ask @sorawit and @evilpeach if you are confused.

### PR DESCRIPTION
**Pull request by tansawit**
Fixes: #1725

### CODE REVIEW

File: scan/src/components/modal/ValidatorStakingInfo.re [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    let unbondingListSub = UnbondingSub.getUnbondingList(delegatorAddress, validatorAddress);
+
+    let%Sub info = infoSub;
+    let%Sub balanceAtStake = balanceAtStakeSub;
+    let%Sub unbonding = unbondingSub;
+    let%Sub unbondingList = unbondingListSub;
+
+    let balanceAtStakeAmount = balanceAtStake.amount->Coin.getBandAmountFromCoin;
+    let unbondingAmount = unbonding->Coin.getBandAmountFromCoin;
+    let rewardAmount = balanceAtStake.reward->Coin.getBandAmountFromCoin;
```
Comment (evilpeach) [inline scan/src/components/modal/ValidatorStakingInfo.re]: Let's change to `let balanceAtStakeAmount = balanceAtStake.amount` and move `Coin.getBandAmountFromCoin` into `stakingBalanceDetail`.

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — DISMISSED** [review]
LGTM

File: scan/src/components/modal/ValidatorStakingInfo.re [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+    </Col>
+  </Row>;
+};
+
+module StakingInfo = {
+  [@react.component]
+  let make = (~delegatorAddress, ~validatorAddress) =>
+    {
+      Js.Console.log(validatorAddress |> Address.toOperatorBech32);
+      Js.Console.log(delegatorAddress |> Address.toBech32);
```
Comment (evilpeach) [inline scan/src/components/modal/ValidatorStakingInfo.re]: remove log

File: scan/src/subscriptions/UnbondingSub.re [hunk 3]
```diff
@@ -14,6 +19,31 @@ module SingleConfig = [%graphql
 |}
 ];
 
+module MultiConfig = [%graphql
+  {|
+  subscription Unbonding($delegator_address: String!, $validator_address: String!) {
+  unbonding_delegations(where: {_and: {delegator_address: {_eq: $delegator_address}, validator_address: {_eq: $validator_address}}}, order_by: {completion_time: desc}) @bsRecord {
```
Comment (evilpeach) [inline scan/src/subscriptions/UnbondingSub.re]: asc

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by prin-r — APPROVED** [review]

**Pull request merged**